cmd/zetaclientd/start.go

```diff
@@ -77,7 +86,7 @@ func Start(_ *cobra.Command, _ []string) error {
 		return errors.Wrap(err, "unable to update app context")
 	}
 
-	log.Info().Msgf("Config is updated from zetacore\n %s", cfg.StringMasked())
+	log.Debug().Msgf("Config is updated from zetacore\n %s", cfg.StringMasked())
```

Comment:
Unrelated change but I don't think logging the whole config every start is great